src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -1051,7 +1051,10 @@ protected void doRun() throws IOException {
                         LOGGER.finer(() -> "Checking EC2 Connection on: " + ec2_cloud.getDisplayName());
                         try {
                             if(ec2_cloud.connection != null) {
-                                ec2_cloud.connection.describeInstances();
+                                List<Filter> filters = new ArrayList<>();
+                                filters.add(new Filter("tag-key").withValues("bogus-EC2ConnectionKeepalive"));
```

Comment:
I don't know if there's a more idiomatic way to craft a filter that returns few/no instances, while also not triggering an `AmazonClientException`.  This was just the first thing that came to mind.